src/npm.ts

```diff
@@ -194,6 +194,7 @@ export async function resolveNpmImports(root: string, path: string): Promise<Imp
   promise = (async function () {
     try {
       const filePath = await populateNpmCache(root, path);
+      if (!/\.(m|c)?js$/i.test(path)) return []; // not JavaScript; TODO traverse CSS, too
```

Comment:
This avoids a spurious “unable to fetch or parse” when `import.meta.resolve` is passed a non-JavaScript file. I’m a little loathe to enumerate the allowed file extensions for JavaScript, but I think this is basically fine and >99% of the time it should be `.js` anyway, and plus we need a supported file extension to serve the MIME type besides. And it’s not like we have a bundler that will support TypeScript or JSX (yet).